docker/ckan/files/ckan.ini

```
@@ -137,7 +137,7 @@ ckan.cors.origin_allow_all = false
 ckan.cors.origin_whitelist =
 
 ## Plugins Settings ############################################################
-ckan.plugins = push_errors unckan image_view pdf_view datastore datatables_view datapusher_plus activity tracking api_tracking superset announcements dbquery selfinfo citeproc
+ckan.plugins = push_errors unckan image_view pdf_view datastore datatables_view datapusher_plus activity tracking api_tracking superset announcements dbquery selfinfo citeproc scheming_datasets scheming_groups scheming_organizations charts_view charts_builder_view
```

Creo que la extension scheming no esta en uso aca. Revisar y confirmar

```suggestion
ckan.plugins = push_errors unckan image_view pdf_view datastore datatables_view datapusher_plus activity tracking api_tracking superset announcements dbquery selfinfo citeproc charts_view charts_builder_view
```